docker/dev/Dockerfile

```diff
@@ -30,7 +30,8 @@ RUN set -x && apt-get update -qq \
   thrift-compiler libthrift-dev \
   postgresql-server-dev-14 && \
   ln -s /usr/bin/gcc-11 /usr/bin/gcc && \
-  ln -s /usr/bin/g++-11 /usr/bin/g++
+  ln -s /usr/bin/g++-11 /usr/bin/g++ && \
+  apt-get install -y postgresql-14
```

Comment:
No, I meant that the installation of the `postgresql-14` package should go to the end of the package list for the existing `apt-get install` command, right after ` postgresql-server-dev-14`.